_episodes/8.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: episode
+title:  "8: Trino: A Ludicrously Fast Query Engine: Past, Present, and Future"
```

Comment:
Use sentence case. ... the rest of the site and docs do as well. Ideally we go back and update all episodes